### ISSUE TITLE
Dvorak keybindings on Ubuntu don't work by default

### ISSUE THREAD

**Issue by jwulf** [Microsoft/vscode#54119]
- VSCode Version: 1.25.0
- OS Version: Ubuntu 18.04

Steps to Reproduce:

1. Use Dvorak keyboard layout
2. Do something like Ctrl-C to copy
3. Observe 'Ctrl-I pressed, waiting for next key in chord'

Workaround described in https://github.com/Microsoft/vscode/issues/23964 (set `keyboard.dispatch: "keyCode"`) works.

**Comment by alexandrudima** [comment]
I have tried to reproduce, but I am failing to configure this correctly on Ubuntu. I've added a new Input Source here:
![image](https://user-images.githubusercontent.com/5047891/43308528-32b2d984-9182-11e8-9310-ecb6ccd1c353.png)

Even after switching the layout from the top bar selector, all applications in my Ubuntu 18 installation still work as if the keyboard layout was QWERTY. Has this changed substantially with Ubuntu 18, I was able to get this to work in the past with Ubuntu 16? :

![kapture 2018-07-27 at 9 50 29](https://user-images.githubusercontent.com/5047891/43308693-a17a0702-9182-11e8-9a92-90fba0de75bd.gif)
